src/Agent.Listener/MessageListener.cs

```diff
@@ -270,12 +270,12 @@ private bool IsGetNextMessageExceptionRetriable(Exception ex)
                 ex is AccessDeniedException ||
                 ex is VssUnauthorizedException)
             {
-                Trace.Info($"Non-retriable exception: {ex.Message}");
+                Trace.Info($"Non-retryable exception: {ex.Message}");
```

Comment:
From a hasty web search, I can point you to three dictionary references for the word "retriable":
 * https://en.wiktionary.org/wiki/retriable
 * http://www.yourdictionary.com/retriable
 * http://www.wordow.com/english/dictionary/?t=retriable

I didn't find any dictionary entries for "retryable" (Urban Dictionary doesn't count).